src/1-theory/3-input-and-output.adoc

```diff
@@ -477,7 +477,7 @@ Content-Type: application/json
 
 {
     "checkin": {
-        "place_id" : 1,
+        "place_id": 1,
         "message": "This is a bunch of text.",
         "with_friends": [1, 2, 3, 4, 5]
     }
```

Comment:
in the paragraph below you say "You know who the user is from their auth token." but there is no token in the examplle